### ISSUE TITLE
-Wimplicit-fallthrough in drivers/hwmon/smpro-hwmon.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1751]
```
drivers/hwmon/smpro-hwmon.c:378:2: error: unannotated fall-through between switch labels [-Werror,-Wimplicit-fallthrough]
        default:
        ^
drivers/hwmon/smpro-hwmon.c:378:2: note: insert 'break;' to avoid fall-through
        default:
        ^
        break;
1 error generated.
```

Patch submitted: https://lore.kernel.org/20221027195238.1789586-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/groeck/linux-staging/c/50ec24a73d1f86986035892acb16d9d67784908a

**Comment by nathanchance** [comment]
Fixed in next-20221031.
